cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -69,7 +71,7 @@
 class InformationMetadata(BaseMetadata):
     role: ClassVar[RoleTypes] = RoleTypes.information_architect
     schema_: SchemaCompleteness = Field(alias="schema")
-    extension: ExtensionCategory | None = ExtensionCategory.addition
```

Comment:
it was not serializing correctly ?

Reply:
I updated pydantic and then it stopped serializing as expected.